openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -9,18 +10,48 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
     order = pyblish.api.IntegratorOrder
     label = "Kitsu Note and Status"
     families = ["render", "kitsu"]
+
+    # status settings
     set_status_note = False
     note_status_shortname = "wfa"
 
+    # comment settings
+    custom_comment_template = {}
+    custom_comment_template = {
+        "enabled": False,
+        "comment_template": "{comment}",
+    }
+
+    def safe_format(self, msg, **kwargs):
+        """If key is not found in kwargs, set None instead"""
+
+        def replace_missing(match):
+            value = kwargs.get(match.group(1), None)
+            if value is None:
+                self.log.warning(
+                    "Key `{}` was not found in instance.data "
+                    "and will be rendered as `` in the comment".format(
+                        match.group(1)
+                    )
+                )
+                return ""
+            else:
+                return str(value)
```

Comment:
Technically the value in `instance.data` could be `None` for something and be valid? :)

```suggestion
            key = match.group(1)
            if key not in kwargs:
                self.log.warning(
                    "Key `{}` was not found in instance.data "
                    "and will be rendered as `` in the comment".format(
                        key
                    )
                )
                return ""
            else:
                return str(kwargs[key])
```

Reply:
Wops 😄